addons/website/static/src/js/website.backendDashboard.js

```diff
@@ -54,8 +54,8 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
     fetch_data: function() {
         var self = this;
         return ajax.jsonRpc('/website/fetch_dashboard_data', 'call', {
-            'date_from': this.date_from.format('YYYY-MM-DD'),
-            'date_to': this.date_to.format('YYYY-MM-DD'),
+            'date_from': this.date_from.year()+'-'+(this.date_from.month()+1)+'-'+this.date_from.date(),
```

Comment:
ping @ged-odoo 

Don't we normally just `.clone().locale(false)` or `.clone().locale('en')` when we want to reset to the "server" locale?